src/EditorFeatures/CSharpTest2/Recommendations/ReadOnlyKeywordRecommenderTests.cs

```diff
@@ -463,26 +463,31 @@ public static void Test()
 
         [Test.Utilities.CompilerTrait(Test.Utilities.CompilerFeature.ReadOnlyReferences)]
         [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
-        public async Task TestRefReadonlyAsParameterModifierInLambdaExpressions()
+        public async Task TestRefReadonlyNotAsParameterModifierInLambdaExpressions()
         {
             await VerifyAbsenceAsync(@"
-public delegate int Delegate(ref readonly int p);
+public delegate int Delegate(ref int p);
 
 class Program
 {
     public static void Test()
     {
-        Delegate lambda = (ref $$ int p) => p;
+        // This is bad. We can't put 'ref $ int p' like in the other tests here because in this scenario:
+        // 'Delegate lambda = (ref r int p) => p;' (partially written 'readonly' keyword),
+        // the syntax tree is completely broken and there is no lambda expression at all here.
+        // 'ref' starts a new local declaration and therefore we do offer 'readonly'.
+        // Fixing that would have to involve either changing the parser or doing some really nasty hacks.
+        // Delegate lambda = (ref $$ int p) => p;
```

Comment:
not sure what to do here

Reply:
file bug.

Reply:
I don't think this is a bug

Reply:
Yeah, I agree with @CyrusNajmabadi. File a bug. We can do a better job with error recovery in the parser here.